### ISSUE TITLE
Updated to latest and commit does not work. Crashes the whole program

### ISSUE THREAD

**Issue by HElkhoury** [desktop/desktop#2930]
### Description
I updated to the latest, part of the auto update. Once i restarted the program, i am no longer able to commit to the web from the desktop. It used to work great prior to the update. No other changes made to the PC. the prompt asks to restart, which i do, then the same issue.

### Version
Version: 1.0.2
OS: Windows 10.0.14393

### Error Code
Error: write EOF
    at exports._errnoException (util.js:1050:11)
    at WriteWrap.afterWrite (net.js:813:14)

**Comment by shiftkey** [comment]
@HElkhoury thanks for the report!

Could you attach the full log files? The **Help** | **Show Logs in Explorer** menu items should open the logs folder, and inside those are the log files generated by day. This will provide some more information to help troubleshoot.

**Comment by HElkhoury** [comment]
Here is the complete log. Now i see it is one of the long filenames. So i will try to fix my specific use case and let you know. But the crashing i believe is an issue.

[2017-09-30.desktop.production.log](https://github.com/desktop/desktop/files/1347275/2017-09-30.desktop.production.log)
[2017-10-01.desktop.production.log](https://github.com/desktop/desktop/files/1347276/2017-10-01.desktop.production.log)

**Comment by shiftkey** [comment]
@HElkhoury thanks for the log files, this is the interesting error:

```
2017-09-30T20:58:31.935Z - error: [main] Error: write EOF
    at exports._errnoException (util.js:1050:11)
    at WriteWrap.afterWrite (net.js:813:14)
2017-09-30T20:58:31.936Z - error: [ui] `git update-index --add --remove --replace -z --stdin` exited with an unexpected code: 128.
error: lstat("@Documentation/Support Documentation/CAN_BOOTLOADER and I2C-CAN Bridge/I2C_CAN_Bridge/I2C_CAN_Bridge.cydsn/DP8051/DP8051_Keil_903/Debug/.deps/KEIL_8051_ASM_FILE.P"): Filename too long
fatal: Unable to process path @Documentation/Support Documentation/CAN_BOOTLOADER and I2C-CAN Bridge/I2C_CAN_Bridge/I2C_CAN_Bridge.cydsn/DP8051/DP8051_Keil_903/Debug/.deps/KEIL_8051_ASM_FILE.P

2017-09-30T20:58:32.303Z - info: [main] Error report submitted
```

Can you confirm these two issues are intertwined? Are you able to make a commit to the repository by changing a file that has a path less than `MAX_PATH` length on Windows?

**Comment by HElkhoury** [comment]
@shiftkey, I tried but did not work. I deleted the files which were listed under the error, and still the same.
I resolved the issue by brute force. I will add a disclaimer that this is my first Github experience. Just got started on it, so i am not an expert, but a student of the training modules.
What i did was deleted the whole repository on this PC, where the problem occured, including the Github desktop client. Reinstalled the client (which came in as 1.0.3, then downloaded the repository from the website onto another PC using a USB stick. I then copied the whole thing again on this PC. Then made a few changes, and committed them to the online repository and they went through.
Not sure if this helps or makes things worse. But the issue has gone away since then and the workflow works on both my PCs now.
